components/email/src/Email.svelte

```diff
@@ -1079,15 +1108,40 @@
                   {thread.snippet}</span
                 >
               </div>
-              {#if show_received_timestamp}
-                <div class="date">
+              <div
+                class:date={show_received_timestamp}
+                class:action-icons={show_thread_actions}
+              >
+                {#if show_received_timestamp}
                   <span>
                     {formatPreviewDate(
                       new Date(thread.last_message_timestamp * 1000),
                     )}
                   </span>
-                </div>
-              {/if}
+                {:else if show_thread_actions}
+                  <!-- Read: ch65594 -->
+                  {#if false}
```

Comment:
same from Oliver: 

the way we would do delete is move the email to a folder with a "trash" label
the only issue is that each email provider has a different label for what "trash" is...
so we'd have to try each provider in the middleware to delete